lib/drives/pages/drive_detail_page.dart

```diff
@@ -238,47 +241,54 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
       final icon = action.isStart ? startIcon : endIcon;
       final profile = action.profile;
 
-      Widget container = Container(
-        decoration: BoxDecoration(
-          color: Theme.of(context).colorScheme.onSurface.withOpacity(0.1),
-          borderRadius: const BorderRadius.all(Radius.circular(5.0)),
-        ),
-        child: Material(
-          color: Colors.transparent,
-          child: InkWell(
-            onTap: () {},
-            child: Padding(
-              padding: const EdgeInsets.symmetric(horizontal: 5.0, vertical: 10.0),
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.spaceAround,
-                children: [
-                  Expanded(
-                    child: Align(
-                      alignment: Alignment.centerLeft,
-                      child: Row(
-                        mainAxisSize: MainAxisSize.min,
-                        children: action.seats <= 2
-                            ? List.generate(action.seats, (index) => icon)
-                            : [
-                                icon,
-                                const SizedBox(width: 2),
-                                Text("x${action.seats}"),
-                              ],
+      Widget container = Semantics(
+        button: true,
+        label:
+            "${action.isStart ? "Pick up" : "Drop-off"} ${action.seats} people associated with ${action.profile.username}.",
```

Comment:
I like this. But it should probably also announce the start/end of the whole drive? Like it currently only says the place name when you start somewhere

Reply:
I don't really get this, I quite like how it announces the time and place and then who is getting picked up and dropped off, like if you just swipe through the elements I think you get a pretty good idea of the drive

Reply:
Yes but for the first one it should say "Start of the Drive" and for the last one "End of the Drive"